### ISSUE TITLE
A template for creating node modules

### ISSUE THREAD

**Issue by egoist** [awesome-templates/template-requests#1]
## What is this template for?

Creating node module:

- Pure JavaScript (i.e. no transpiler)
- Unit Testing (Jest)
- Linting (XO) + Prettier + Husky + lint-staged

## Are you willing to create this template?

Yes but if you wanna work on this, let me know!

**Comment by 2nthony** [comment]
🤔https://github.com/saojs/sao-nm

**Comment by egoist** [comment]
@evillt yeah but I want to take advantage of GitHub's built-in template repository feature.
